app/src/features/admin/users/AccessRequestList.tsx

```diff
@@ -97,7 +97,6 @@ const AccessRequestList: React.FC = () => {
               <TableCell>Regional Offices</TableCell>
               <TableCell>Request Date</TableCell>
               <TableCell>Status</TableCell>
-              <TableCell></TableCell>
```

Comment:
I think this might still be needed?

There are 7 columns in the data row, the 7th being for the 'Review' button which doesn't get an actual header, but might still need the placeholder to block the space?

Reply:
Ya it is (tested locally)